README.md

```diff
@@ -1,16 +1,43 @@
 # Northcoders News API
 
+## Project Summary
+
+This app enables you to search through a PostgreSQL database of news articles, topics, authors and associated comments. Functionality includes (please seen "endpoints.json" for a detailed list):
+
+- Requesting all articles, topics or users
+- Requesting individual articles by id
+- Posting or deleting comments
+- Updating properties for a specified article
+
+## Hosted Version
+
+https://nc-news-6eit.onrender.com
+
 ## Setup Instructions
 
+### Requirements
+
+- Node v20.18.0
+- PSQL v12.20
+
+### Initial Setup
+
 This repo contains two databases - one for developer demo data, and another for a smaller set of test data. In order to set the correct environment (either test or developer), you will need to create two separate `.env` files in the root folder:
+
 `.env.test` - in this file, add `PGDATABASE=nc_news_test`
```

Comment:
It'd be better if you were more vague about what the database names are, the reason we don't push the .env files is so our database name stays hidden. You could put something like "database_name_test"